lib/tapioca/helpers/config_helper.rb

```diff
@@ -0,0 +1,85 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "yaml"
+
+module Tapioca
+  module ConfigHelper
+    extend T::Sig
+
+    sig { returns(String) }
+    attr_reader :command_name
+
+    sig { params(args: T.untyped, local_options: T.untyped, config: T.untyped).void }
+    def initialize(args = [], local_options = {}, config = {})
+      # Store current command
+      command = config[:current_command]
+      command_options = config[:command_options]
+      @command_name = T.let(command.name, String)
+      @merged_options = T.let(nil, T.nilable(Thor::CoreExt::HashWithIndifferentAccess))
+      @defaults = T.let(nil, T.nilable(Thor::CoreExt::HashWithIndifferentAccess))
+
+      # Filter command options unless we are handling the help command.
+      # This is so that the defaults are printed
+      filter_defaults(command_options) unless command_name == "help"
+
+      # Call original initialize method
+      super
```

Comment:
Dr Obvious is obvious 😂 

```suggestion
      super
```